pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -111,6 +112,9 @@ stdenv.mkDerivation rec {
     # needed for GSETTINGS_SCHEMAS_PATH
     glib gsettings-desktop-schemas gtk3
 
+    # needed for GSSAPI authentication (Kerberos)
+    libkrb5
+
```

Comment:
Adding it to `deps` should already be sufficient, no need for adding to `buildInputs`. Apart from this LGTM.

Reply:
Tested and fixed it, although as a Nix noob I'm not sure what the role of `buildInputs` are if not for runtime dependencies (it looks like that's what `rpath` does in this context). In this context, are `buildInputs` just for making environment variables known to Brave?

Reply:
In most cases buildInputs are for the compilers, while for this binary package there is no compiler involved, but the deps are directly used to construct rpath.